app/code/UAE/GroupedProduct/view/frontend/web/js/bottom-bar.js

```diff
@@ -0,0 +1,69 @@
+/**
+ * @copyright   Copyright (c) 2018 Classy Llama Studios, LLC
+ */
+
+define([
+    'jquery',
+    'jquery/ui'
+], function ($) {
+    'use strict';
+
+    $.widget('bottom.Bar', {
+        options: {
+            backToTop: '.back_to_top',
+            addToCart: '#product-addtocart-button-bottom',
+            addToCartForm: '#product_addtocart_form',
+            qtyInput: '.qty',
+            scrollSpeed: '300'
+        },
+
+        /**
+         * @private
+         */
+        _create: function () {
+            this._clickTop();
+            this._clickAddToCart();
+        },
+
+        /**
+         * Click on the button.
+         *
+         * @private
+         */
+        _clickTop: function () {
+            $(document).on('click', this.options.backToTop, this._animateScroll.bind(this));
+        },
+
+        /**
+         * Scroll to top.
+         *
+         * @private
+         */
+        _animateScroll: function () {
+            $('html, body').animate({
+                scrollTop:0
+            }, this.options.scrollSpeed);
+        },
+
+        /**
+         * Click add to cart button.
+         *
+         * @private
+         */
+        _clickAddToCart: function () {
+            $(document).on('click', this.options.addToCart, this._submitForm.bind(this));
+        },
```

Comment:
@Andrewb89 - Note that with the approach I described about using CSS for div.box-tocart, this _clickAddToCart() function and _submitForm() would not be needed anymore, which would be another advantage, since it would make the code simpler.